specification/servicebus/resource-manager/Microsoft.ServiceBus/2017-04-01/servicebus.json

```diff
@@ -3726,6 +3922,10 @@
             },
             "partnerNamespace": {
               "type": "string",
+              "description": "ARM Id of the Primary/Secondary eventhub namespace name, which is part of GEO DR pairning"
+            },
+			"alternateName": {
```

Comment:
Could you please update the example (or add an example) with this field?
Thanks,

Reply:
updated the Example with alternateName parameter.